projects/storefrontstyles/scss/components/myaccount/_my-coupons.scss

```diff
@@ -0,0 +1,55 @@
+%cx-my-coupons {
+  .cx-coupon-deck {
+    padding: 1.25rem 0 0 0;
+  }
+
+  .cx-coupon-spinner {
+    padding: 5rem 0 5rem 0;
+  }
+
+  .cx-coupon-card {
+    padding-bottom: 30px;
+  }
+
+  .cx-my-coupons-form-group {
```

Comment:
For all css property values below this line:
We use CSS Variables only for colors, therefore the rest of all CSS properties should have regular values.

Reply:
updated.